### ISSUE TITLE
Rename root folder

### ISSUE THREAD

**Issue by Bengt** [atom/tree-view#179]
tree-view can currently not rename the opened root folder. When I click the root folder in the tree view and press `F2` or select `right-click` - `Rename` nothing happens.

**Comment by alexchandel** [comment]
This is still an issue in 1.36.1, and it shouldn't be.  Renaming the root folder is easily something Atom could do.

**Comment by aldevel** [comment]
Yep, seemingly this very basic feature is still missing for a while now. Probably it's feasible by removing it obviously not from Atom and somehow making Atom to forget about it what I'm just trying to figure out somehow but it's definitely weird not having this option by default for a very long time now.
